src/main/resources/assets/jade/lang/ko_kr.json

```diff
@@ -95,30 +202,114 @@
+	"config.jade.plugin_minecraft.next_entity_drop_extra_msg": "달걀,인갑",
 	"jade.nextEgg": "다음 달걀: %s",
+	"jade.nextScute": "다음 인갑: %s",
 	"config.jade.plugin_minecraft.breaking_progress": "부수기 진행률",
 	"config.jade.plugin_minecraft.command_block": "명령 블록",
 	"config.jade.plugin_minecraft.harvest_tool": "수확 도구 표시",
 	"config.jade.plugin_minecraft.harvest_tool.new_line": "새로운 줄에 도구 표시",
 	"config.jade.plugin_minecraft.harvest_tool.effective_tool": "효과적인 도구 표시",
+	"config.jade.plugin_minecraft.harvest_tool.show_unbreakable": "\"부서지지 않음\" 표시",
+	"config.jade.plugin_minecraft.harvest_tool.creative": "창조 모드에서 표시",
+	"config.jade.plugin_minecraft.harvest_tool.creative_desc": "창조 모드나 관전자 모드에서도 수확 정보를 표시",
 	"jade.harvest_tool.unbreakable": "부서지지 않습니다",
 	"config.jade.plugin_minecraft.enchantment_power": "부여력",
+	"config.jade.plugin_minecraft.enchantment_power_desc": "블록의 부여력을 표시합니다.",
 	"config.jade.plugin_minecraft.total_enchantment_power": "총 부여력",
 	"jade.ench_power": "부여력: %s",
 	"config.jade.plugin_minecraft.player_head": "플레이어 머리",
 	"config.jade.plugin_minecraft.profession": "주민 직업",
 	"config.jade.plugin_minecraft.item_tooltip": "아이템 설명",
+	"config.jade.plugin_minecraft.item_storage": "아이템 저장소",
+	"config.jade.plugin_minecraft.item_storage_extra_msg": "inventory,chest",
+	"config.jade.plugin_minecraft.item_storage.detailed_amount": "상세 수량",
+	"config.jade.plugin_minecraft.item_storage.detailed_amount_desc": "[상세 보기] 키를 눌렀을 때 표시할 아이템 수량.",
+	"config.jade.plugin_minecraft.item_storage.normal_amount": "일반 수량",
+	"config.jade.plugin_minecraft.item_storage.normal_amount_desc": "[상세 보기] 키를 누르지 않았을 때 표시할 아이템 수량.",
+	"config.jade.plugin_minecraft.item_storage.show_name_amount": "이름 표시 임계값",
+	"config.jade.plugin_minecraft.item_storage.show_name_amount_desc": "아이템 아이콘 대신 이름을 표시할 최대 아이템 수량.",
+	"config.jade.plugin_minecraft.item_storage.items_per_line": "한 줄당 아이템 개수",
 	"jade.locked": "저장소가 잠겨있습니다",
+	"jade.loot_not_generated": "아직 전리품이 생성되지 않았습니다",
+	"config.jade.plugin_minecraft.energy_storage": "에너지 저장소",
+	"config.jade.plugin_minecraft.energy_storage.detailed": "상세-전용",
+	"config.jade.plugin_minecraft.energy_storage.detailed_desc": "[상세 보기] 키를 눌렀을 때만 에너지 정보를 표시.",
 	"jade.fe": "%s / %s",
+	"config.jade.plugin_minecraft.fluid_storage": "액체 저장소",
+	"config.jade.plugin_minecraft.fluid_storage_extra_msg": "tank,liquid",
+	"config.jade.plugin_minecraft.fluid_storage.detailed": "상세-전용",
+	"config.jade.plugin_minecraft.fluid_storage.detailed_desc": "[상세 보기] 키를 눌렀을 때만 액체 정보를 표시.",
 	"jade.fluid": "%s %s",
+	"jade.fluid2": "%s %s / %s",
 	"jade.fluid.empty": "비어있음",
 	"config.jade.plugin_minecraft.lectern": "독서대",
+	"config.jade.plugin_minecraft.animal_owner": "동물 주인",
+	"config.jade.plugin_minecraft.animal_owner_extra_msg": "horse,pet",
+	"jade.owner": "주인: %s",
+	"jade.infinity": "+∞",
+	"jade.seconds": "{0}초",
+	"jade.minutes": "{0}m",
+	"jade.minutes_seconds": "{0}m {1}s",
+	"jade.error": "<%s 모드에서 오류 발생, 보고해주세요>",
```

Comment:
Please explicitly inform the user to report to the appropriate mod developer, i.e., translate the latest version as is